test/base/ModelerSpec.js

```diff
@@ -84,6 +86,18 @@ describe('<BaseModeler>', function() {
       ...options
     });
 
+    modeler.on('commandStack.changed', function() {
+
+      setTimeout(function() {
+
+        modeler.saveXML({ format: true }).then(function(result) {
+          console.clear();
```

Comment:
Clearing the console also swallows all potential errors. I'd say we're fine without it.

Reply:
Removed the clear portion.